packages/evm/src/utils/depositFns.ts

```diff
@@ -65,6 +65,7 @@ export const executeDeposit = async (
   overrides?: ethers.PayableOverrides,
 ): Promise<PopulatedTransaction> => {
   const transactionSettings = {
+    // TODO: change this based on "twap"
```

Comment:
is this still applicable?

Reply:
I have to consult with the solidity team regarding this one